neat-form-core/src/test/java/com/nerdstone/neatformcore/robolectric/builders/CheckBoxViewBuilderTest.kt

```diff
@@ -33,12 +33,13 @@ class `Test building CheckBox view` : BaseJsonViewBuilderTest(){
     }
 
     @Test
-    fun `Should set text on checkbox`() {
+    fun `Should set text and textSize on checkbox`() {
         val text = "Am a checkbox"
-        viewProperty.viewAttributes = hashMapOf("text" to text)
+        val checkBoxTextSize = 14
+        viewProperty.viewAttributes = hashMapOf("text" to text,"check_box_text_size" to checkBoxTextSize)
```

Comment:
Remember to change this **check_box_text_size** if you change the attribute name.